carla/evaluation/benchmark.py

```diff
@@ -63,7 +64,12 @@ def compute_ynn(self) -> pd.DataFrame:
             self._factuals, self._counterfactuals
         )
 
-        ynn = yNN(counterfactuals_without_nans, self._recourse_method, self._mlmodel, 5)
+        if counterfactuals_without_nans.shape[0] == 0:
+            ynn = np.nan
```

Comment:
Pandas also has a nice .empty function, that would semantically be a bit cleaner, no blocker though

Reply:
👍